website/content/en/docs/building-operators/ansible/testing-guide.md

```diff
@@ -20,13 +20,16 @@ To begin, you sould have:
 
     ```
     .
-    ├── build/
-    ├── deploy/
-    ├── molecule/
-    ├── roles/
-    ├── playbook.yml (optional)
+    ├── config
+    ├── Dockerfile
+    ├── Makefile
+    ├── molecule
+    ├── playbooks
+    ├── PROJECT
     ├── requirements.yml
+    ├── roles
     └── watches.yaml
+
```

Comment:
The structure of the molecule changes a lot. Requires more updates here.

Reply:
Actually, I'm realizing I don't know enough about this to update it. I'll add a TODO.